pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -174,6 +174,7 @@ type TidbClusterSpec struct {
 
 	// Affinity of TiDB cluster Pods
 	// +optional
+	// +k8s:openapi-gen=false
```

Comment:
Since no description now, we can revert the changes in *types.go.

Reply:
updated.